### DISCUSSION TITLE
Light wallet macOS - unable to cancel offer

### DISCUSSION THREAD

**Discussion by hajes** [Chia-Network/chia-blockchain-gui#1371] [General]
### What happened?

I have created two offers, but there seems to be some weird [bug](https://github.com/Chia-Network/chia-blockchain-gui/issues/1310). 2 x 3 XCH offers reserve >50XCH for some unknown reason.

I tried to cancel three times, but it is still there even after re-syncing wallet. I have even put fee 0.0001 to cancel...both offers stuck somewhere, and my wallet balance is not spendable???

there is some weird error in log.

### Version

1.6.2

### What platform are you using?

macOS

### What ui mode are you using?

GUI

### Relevant log output or stacktrace

```shell
wallet chia.rpc.rpc_server        : WARNING  Error while handling message: Traceback (most recent call last):
  File "chia/rpc/rpc_server.py", line 329, in safe_handle
  File "chia/rpc/rpc_server.py", line 320, in ws_api
  File "chia/rpc/wallet_rpc_api.py", line 1628, in cancel_offer
  File "chia/wallet/trade_manager.py", line 247, in cancel_pending_offer_safely
  File "chia/wallet/wallet.py", line 483, in generate_signed_transaction
  File "chia/wallet/wallet.py", line 337, in _generate_unsigned_transaction
ValueError: Can't spend more than wallet balance: 880572769848 mojos
```

**Comment by hajes** [comment]
it can be cancelled only without `fee sending to the blockchain`.

**Comment by BrandtH22** [comment]
@hajes , based on your messages in #1370 it appears this has been resolved. 
Feel free to reopen this ticket if that is not the case or join our discord (https://discord.gg/chia)
